pygmt/datasets/load_remote_dataset.py

```diff
@@ -226,6 +226,26 @@ class GMTRemoteDataset(NamedTuple):
             "03m": Resolution("03m", registrations=["gridline"], tiled=True),
         },
     ),
+    "venus_relief": GMTRemoteDataset(
+        title="Venus relief",
+        name="venus relief",
+        long_name="Venus relief",
```

Comment:
For the datasets of the Earth, underscores are used for `name`.

```suggestion
        title="Venus relief",
        name="venus_relief",
        long_name="Venus relief",
```